dbt/adapters/athena/impl.py

```diff
@@ -722,3 +722,24 @@ def get_columns_in_relation(self, relation: AthenaRelation) -> List[AthenaColumn
         return [
             AthenaColumn(column=c["Name"], dtype=c["Type"], table_type=table_type) for c in columns + partition_keys
         ]
+
+    @available
+    def delete_from_glue_catalog(self, relation: AthenaRelation):
```

Comment:
this looks good, could you please add some unit test please? have a look at the test suite here: https://github.com/dbt-athena/dbt-athena/blob/main/tests/unit/test_adapter.py

Reply:
Yes, I'll give that a try.